src/components/pill-selector-dropdown/__tests__/PillSelectorDropdown.test.js

```diff
@@ -45,17 +45,6 @@ describe('components/pill-selector-dropdown/PillSelectorDropdown', () => {
             expect(selectorDropdown.contains(children)).toBe(true);
         });
 
-        test('should render pill selector', () => {
```

Comment:
This test was not only causing some problems, but is also unnecessary as this functionality is already tested in `SelectorDropdown.test.js`

Reply:
Curious on what problems this was causing?

Reply:
The test didn't seem to work and I couldn't wrap my head around it and then I realized that it was already tested in the SelectorDropdown so it didn't seem necessary to continue. 

However, I just took a second look at it. I found a bug in this test which I didn't realize at first and it was causing me headaches. It showed up after modifying PIllSelector to return a forwardRef component. I just fixed the bug and added the test back in there.

Reply:
I guess it was a good exercise!